libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -804,6 +805,21 @@ static DNNReturnType execute_model_tf(const DNNModel *model, const char *input_n
             out_frame->width = output[0].width;
             out_frame->height = output[0].height;
         }
+        break;
+    case DFT_ANALYTICS_DETECT:
+        if (!model->detect_post_proc) {
+            av_log(ctx, AV_LOG_ERROR, "detect filter needs to provide post proc\n");
+            return DNN_ERROR;
+        }
+        model->detect_post_proc(out_frame, output, nb_output, model->filter_ctx);
```

Comment:
does your detection work without this patch?

Reply:
Yes, I tested tf detection with a picture of dog, and it works well with all my patches applied.

Reply:
we need to merge part of this change into others, and in the last patch to enable tf+detect, we'd also add detail command lines.

Reply:
OK, sure